### PR TITLE
Remove columnar/.gitignore

### PR DESCRIPTION
**Pull request by onurctirtir**
Since we already ignore them by other .gitignore files in this repo

### CODE REVIEW

File: src/backend/columnar/.gitignore [hunk 1]
```diff
@@ -1,68 +0,0 @@
...
-/results/*
-/.deps/*
-/regression.diffs
-/regression.out
-.vscode
-
-*.pb-c.*
-
-# ignore files that could be created by circleci automation
-files.lst
```
Comment (SaitTalhaNisanci) [inline src/backend/columnar/.gitignore:66]: Is this ignored by any other gitignore?  

It seems like this file isn't covered entirely by other git files( with a quick look). Have you checked the rules and realized this is not necessary?

Comment (onurctirtir) [reply src/backend/columnar/.gitignore:66]: I've seen that some of the lines (like you shared) were not covered in other gitignore files.
Then installed citus/executed regression tests etc. and verified that there were no such junk files, so I thought that they might be left over from old cstore repo or something

Comment (SaitTalhaNisanci) [reply src/backend/columnar/.gitignore:66]: They might be leftovers or they might be files that are generated after some runs, or in certain environments. It is a bit hard to be sure, I just wanted to point out this possibility.

Comment (onurctirtir) [reply src/backend/columnar/.gitignore:66]: > It is a bit hard to be sure, I just wanted to point out this possibility.

yeah that makes sense, let's wait for @jeff-davis's input then

**Review by jeff-davis — APPROVED** [review]

**Pull request merged**